com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalTarget.cs

```diff
@@ -369,6 +369,36 @@ static class CorePasses
             customInterpolators = CoreCustomInterpDescriptors.Common
         };
 
+        public static readonly PassDescriptor DepthNormalOnly = new PassDescriptor()
```

Comment:
Is there any changes here  other than moving the code around?

Reply:
No, the code was moved around from Lit to Core so we could have Unlit ShaderGraph shaders w/ a depthnormals pass.